code/controllers/subsystem/SSticker.dm

```diff
@@ -226,7 +226,19 @@ SUBSYSTEM_DEF(ticker)
 	mode.pre_pre_setup()
 	var/can_continue = FALSE
 	can_continue = mode.pre_setup() //Setup special modes. This also does the antag fishing checks.
+
+	// Now that every other piece of the round has initialized, lets setup player job scaling // Not completely true anymore, but if we do this later we only get the latejoin slots after the round has started
```

Comment:
There's no need to have a comment commenting about how another comment is wrong. 
```suggestion
	// Enable highpop slots just before we distribute jobs.
```